src/trix/controllers/attachment_editor_controller.js

```diff
@@ -148,6 +148,8 @@ export default class AttachmentEditorController extends BasicObject {
       withCallback: this.didClickActionButton,
     })
 
+    triggerEvent("trix-attachment-before-toolbar", { onElement: this.element, attributes: { toolbar: element, attachment: this.attachment } } )
```

Comment:
`trix-attachment-before-toolbar` is a bit of a mouthful. It follows the existing `trix-attachment-*` pattern of the other events, but if we think of something better we can change it.

Reply:
I was going to suggest `trix-attachment-before-toolbar-added` :) So I think it's fine.

There's one extrac space between the last `} )`, though.

Reply:
Ah so there is! That's fixed now.